BrainPortal/cbrain_plugins/cbrain-plugins-base/cbrain_task/boutiques_task/views/_show_params.html.erb

```diff
@@ -24,12 +24,24 @@
 
 <%-
   # Parameter groups
-  @descriptor = @task.descriptor_for_show_params
-  inputs      = @descriptor.inputs
-  file_inputs = @descriptor.file_inputs
-  params      = inputs.select { |i| i.type != 'File' }
-  outputs     = @descriptor.output_files
+  @descriptor     = @task.descriptor_for_show_params
+  inputs          = @descriptor.inputs
+  file_inputs     = @descriptor.file_inputs
+  file_inputs_ids = file_inputs.map(&:id)
+  params          = inputs.select { |i| i.type != 'File' }
+  param_ids       = params.map { |p| p.id }
+  outputs         = @descriptor.output_files
+  groups          = @descriptor.groups
+
+  # Find parameters that are part of groups
+  param_with_groups_ids    = groups.map { |m| m.members || []}.flatten - file_inputs.map(&:id)
```

Comment:
This is super confusing. What do these variables contain? By reading it is seems param_with_group_ids does not contain a param, does not contain group_ids, it contains param_ids ? Please consider renaming things properly so that a programmer reading your code for the first time know what it is you are manipulating!